README.rst

```diff
@@ -294,15 +294,17 @@ Starts listening structure changes and passes the changes to provided callback f
 
         node.subscribe_to_structure_changes(on_change)
 
-node.subscribe_to_value_changes(callback)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+node.subscribe_to_value_changes(callback, fs=5)
```

Comment:
Our C++ client uses separate `publishingFs` and `samplingFs` parameters. Perhaps expose both right away?

``` 
  /**
   * @brief Subscribe to the value changes of this node.
   *        The node must have a defined value type. If the remote node value
   *        changes, the provided listeners' valueChanged methods are called
   *        with this node as the argument.
   * @param publishingFs  Maximum frequency that value updates are expected (controls how many changes
   *                      are sent in a single packet).
   * @param listener  IValueListener implementation that handles callbacks.
   * @param samplingFs  Maximum amount of value updates sent per second (controls the amount of data transfered).
   */
  virtual void SubscribeToValueChanges(double publishingFs, IValueListener* listener, double samplingFs = 0) = 0;
```

At the very least the naming scheme should take into account that there might be too kinds of fs in the future. Because renaming arguments can break code in python. So I'd rename `fs` to `publishing_fs`. Would be more consistent also with C++ client.